cli/src/main.rs

```diff
@@ -136,13 +140,45 @@ pub async fn main() {
       .about("Deploys your repository to an existing app hosted in one of the deploy profiles from anycloud.json")
       .arg_from_usage("<APP_ID> 'Specifies the alan app to upgrade'")
       .arg_from_usage("-e, --env-file=[ENV_FILE] 'Specifies an optional environment file relative path'")
+    )
+    .subcommand(SubCommand::with_name("config")
+      .about("Manage Deploy Configs used by apps from the anycloud.json in the current directory")
+      .setting(AppSettings::SubcommandRequiredElseHelp)
+      .subcommand(SubCommand::with_name("add")
+        .about("Add a new Deploy Config to the anycloud.json in the current directory")
```

Comment:
This will create a new `anycloud.json` if does not exist, right?

Reply:
yes!